clients/src/test/java/org/apache/kafka/common/config/FileConfigProviderTest.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.kafka.common.config;
 
+import org.apache.kafka.common.config.provider.FileConfigProvider;
```

Comment:
This `FileConfigProviderTest` should be moved into the `config.provider` package.